analytics/build.gradle

```diff
@@ -11,8 +11,6 @@ android {
     defaultConfig {
         minSdkVersion rootProject.androidMinSdk
         targetSdkVersion rootProject.androidTargetSdk
-        versionCode 1
```

Comment:
These were automatically remove by the migration, as I guess they aren't needed for libraries any longer (which makes sense).